guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,517 @@
 {
```

Comment:
<p>Protocol Buffer Format - Have we merged this already?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<ul><li>I misinterpreted a PR and talked to Andrew. I removed this release note.</li></ul>